### PR TITLE
Add list of publications

### LINKED ISSUE #1026: [Suggestion] Create a single page of the resources and publications produced by STAG
Description: I get asked a lot is there a single page that contains links to all the works produced by the STAG.  Every time i need to hunt through the repo to find them and each time i know i'm missing some.

Impact: significantly reduce the Level of Effort by community members in discovering cloud native security publications. It currently requires a community member that has been around through all those publications to pull them together, as we experience turnover and age-off in the knowledge of our community members, we will lose visibility into these works' existence.

Scope: medium - create a PR that captures the below links with a short description of each and take a cursory pass through closed issues with related tags and the content format links (markdown | PDF )

Additional info:

- https://github.com/cncf/tag-security/raw/main/supply-chain-security/supply-chain-security-paper/CNCF_SSCP_v1.pdf
- https://github.com/cncf/tag-security/blob/main/supply-chain-security/supply-chain-security-paper/secure-supply-chain-assessment.md
- https://github.com/cncf/tag-security/blob/main/supply-chain-security/supply-chain-security-paper/sscsp.md
- https://github.com/cncf/tag-security/raw/main/supply-chain-security/secure-software-factory/Secure_Software_Factory_Whitepaper.pdf
- https://www.cncf.io/wp-content/uploads/2022/06/CNCF_cloud-native-security-whitepaper-May2022-v2.pdf
- https://github.com/cncf/tag-security/blob/main/security-lexicon/cloud-native-security-lexicon.md
- https://github.com/cncf/tag-security/tree/main/usecase-personas
- https://github.com/cncf/tag-security/tree/main/policy
- https://github.com/cncf/tag-security/blob/main/security-whitepaper/secure-defaults-cloud-native-8.md
- https://github.com/cncf/tag-security/blob/main/cloud-native-controls/phase-one-announcement.md

### PR DESCRIPTION
**Pull request by szh**
Signed-off-by: Shlomo Heigh <shlomo.heigh@cyberark.com>

Resolves #1026

### CODE REVIEW

**Review by mnm678 — APPROVED** [review]

File: PUBLICATIONS.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+## Secure Defaults: Cloud Native 8
+
+- [Markdown](https://github.com/cncf/tag-security/blob/main/security-whitepaper/secure-defaults-cloud-native-8.md)
+
+## Cloud Native Security Controls Catalog
+
+Mapping of Cloud Native Security Whitepaper and Software Supply Chain Best Practices Paper to NIST SP800-53r5
+
+- [Markdown](https://github.com/cncf/tag-security/blob/main/cloud-native-controls/phase-one-announcement.md)
+- [Spreadsheet](https://docs.google.com/spreadsheets/d/1GUohOTlLw9FKUQ3O23X7ypvJLXN-B3veJGe6YE6JYfU/edit?usp=sharing)
```
Comment (ragashreeshekar) [inline PUBLICATIONS.md:66]: Thanks @szh for this PR. Much appreciated. 
Please incorporate the suggestions in this comment to reference some more efforts from the community 
https://github.com/cncf/tag-security/issues/1026#issuecomment-1382619123

**Review by ragashreeshekar — CHANGES_REQUESTED** [review]
Thanks @szh for this PR. Much appreciated.
Please incorporate the suggestions in this comment to reference some more efforts from the community
https://github.com/cncf/tag-security/issues/1026#issuecomment-1382619123

**Comment by szh** [comment]
@ragashreeshekar I added the translations of the CNSWP and the security assessments. #110 seems to be a bit broader than this - it also wants a broader overview of the TAG and a static site. Is that the direction we want to go in?

**Comment by ragashreeshekar** [comment]
> @ragashreeshekar I added the translations of the CNSWP and the security assessments. #110 seems to be a bit broader than this - it also wants a broader overview of the TAG and a static site. Is that the direction we want to go in?

Thanks @szh, this addresses the requirements of central repo of the TAG contributions. LGTM. 
Microsite from https://github.com/cncf/tag-security/issues/110 is outside of the scope of https://github.com/cncf/tag-security/issues/1026.

**Review by ragashreeshekar — APPROVED** [review]

**Pull request merged**